.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb"
+    ],
+    "globals": {
```

Comment:
what is this globals declaration for? did someone add this intentionally?

Reply:
Not intentionally, I ran `npx eslint --init` to setup eslint. Must be the result of one of the config options I sent. I can remove for now and see if it changes anything?